.github/workflows/tests-cli.yml

```diff
@@ -70,18 +70,22 @@ env:
   # report to tinybird if executed on master
   TINYBIRD_PYTEST_ARGS: "${{ github.ref == 'refs/heads/master' && '--report-to-tinybird ' || '' }}"
 
+permissions: 
+  contents: read # checkout the repository
+
 jobs:
   cli-tests:
     runs-on: ubuntu-latest
     strategy:
       fail-fast: false
       matrix:
         python-version: [ "3.8", "3.9", "3.10", "3.11", "3.12" ]
+        os: [ubuntu-latest, windows-latest]
     timeout-minutes: 10
     env:
       # Set job-specific environment variables for pytest-tinybird
-      CI_JOB_NAME: ${{ github.job }}-${{ matrix.python-version }}
-      CI_JOB_ID: ${{ github.job }}-${{ matrix.python-version }}
+      CI_JOB_NAME: ${{ github.job }}-${{ matrix.python-version }}-${{matrix.os}}
+      CI_JOB_ID: ${{ github.job }}-${{ matrix.python-version }}-${{matrix.os}}
```

Comment:
We do test the compatibility with Windows in `localstack/localstack-cli`. This pipeline is explicitly meant to test the compatibility with the different supported python versions.